src/main/webapp/app/exam/manage/exams/exam-update.component.html

```diff
@@ -139,11 +128,65 @@ <h4>{{ 'artemisApp.examManagement.editExam' | artemisTranslate }}</h4>
                     <input id="numberOfExercisesInExam" name="numberOfExercisesInExam" class="form-control" type="number" min="0" [(ngModel)]="exam.numberOfExercisesInExam" />
                 </div>
             </div>
-
             <div class="form-check mb-3">
-                <input class="form-check-input" type="checkbox" id="randomizeExerciseOrder" name="randomizeExerciseOrder" [(ngModel)]="exam.randomizeExerciseOrder" />
-                <label class="form-check-label" for="randomizeExerciseOrder" jhiTranslate="artemisApp.examManagement.randomizeQuestionOrder"> Randomize Question Order </label>
+                <input class="form-check-input" type="checkbox" id="DublerandomizeExerciseOrder" name="DublerandomizeExerciseOrder" [(ngModel)]="exam.randomizeExerciseOrder" />
+                <label class="form-check-label" for="DublerandomizeExerciseOrder" jhiTranslate="artemisApp.examManagement.randomizeQuestionOrder"> Randomize Question Order </label>
+            </div>
+            <!-- Section exam assessment and student review
+            Deactivated for TestExams, as results are provided right after a TestExam run and complaints are not possible  -->
```

Comment:
Btw, thanks for including comments to make separation visible in the HTML file! I love it!